eng/common/pipelines/templates/steps/validate-all-packages.yml

```diff
@@ -10,18 +19,36 @@ steps:
       displayName: "Set as release build"
       condition: and(succeeded(), eq(variables['SetAsReleaseBuild'], ''))
 
+    - pwsh: |
```

Comment:
Will this ever be in the same job as the create-apiview.yml steps? I'm just trying to ensure we don't end up with duplicate or conflicting variables.

Reply:
The problem here is that something like the create-apiview.yml is used in multiple places, the build job and the release job. It's safer and less persnickety to just have this be in their respective templates.